### DISCUSSION TITLE
Account restricted for violation of terms of service

### DISCUSSION THREAD

**Discussion by wSakuraa** [codewars/codewars.com#3098] [Q&A] [answered]
Hello, sorry to bother you. My English is not very good, so I use a translator to express it. I hope it won't cause any trouble to you. I am a junior student. Because the teacher needs to use codewars to complete the task, the teacher will lead us to complete the code together during class. In fact, we only need to paste it to complete the task and complete the experimental report and hand it to the teacher. It may be that my account has been restricted due to improper operation. In order to complete subsequent learning tasks, is there any way to remove the restriction? Thank you so much.

**Comment by VanessaGreene** [comment, marked as answer]
Hi @wSakuraa, can you please ask your teacher to email vanessa.greene@andela.com about this? I want to help them find a better solution for your class activities on Codewars 😃 

Can you also email info@codewars.com with your username so we can look into your account restriction?
